c7n/actions/securityhub.py

```diff
@@ -236,6 +286,13 @@ def get_finding(self, resources):
             finding_resources.append(self.format_resource(r))
         finding["Resources"] = finding_resources
         finding["Types"] = list(self.data["types"])
+
+        if existing_finding_id:
```

Comment:
this logic is bit problematic, it won't ever tag a resource unless it already has a finding id, which it won't have if the resource is never tagged.